docs/howtos/Client Generation/06types.mdx

```diff
@@ -793,6 +793,197 @@ public final class Animal implements JsonSerializable<Animal> {
 </TabItem>
 </Tabs>
 
+### Discriminator
```

Comment:
Could you also add some description/case of multiple-level discriminator?

Reply:
we kind of prefer that we do this in a separate PR.